.github/ISSUE_TEMPLATE/internal_task.yml

```diff
@@ -1,6 +1,7 @@
-name: Task (Internal Use Only)
-description: Tracking our work through tasks
+name: Internal - Task
+description: Describes an individual task that needs to be completed
 title: "[Task]: "
+labels: ["project: grants.gov"]
```

Comment:
Thanks for adding this, it is tedious of course to add every time. I imagine we could revisit in the future if necessary if other projects need to use the internal task.

Reply:
+++